### ISSUE TITLE
Results list BLAST IDs instead of sequence names

### ISSUE THREAD

**Issue by j-kominek** [wurmlab/sequenceserver#305]
Hi, we deployed SequenceServer at my institute, and it works nicely, with the following issue: the hit list contains BLAST ordinal IDs (e.g. gnl|BL_ORD_ID|11) instead of the actual sequence names. When you get to the actual hits, the sequence names are reported but again below the BLAST IDs. A similar issue has been known in some earlier version of BLAST, but apparently it was fixed some time ago. Moreover, when I'm running a local blast against those same BLAST database files, it reports the sequence names properly, so I'm in a pickle as to why it happens with SequenceServer. Any ideas on how to fix that? Thanks!

**Comment by yeban** [comment]
My guess is your databases were created without `-parse_seqids` option.

**Comment by j-kominek** [comment]
I did try that too, and it didn't seem to make any difference. I can try again to double-check though.

**Comment by yeban** [comment]
I am sorry I was unable to follow up. I wasn't well.

Did you manage to figure this out? What's your BLAST version? Can you post me one sequence out of your FASTA file for which ordinal IDs are returned so I can debug this?

**Comment by j-kominek** [comment]
Hi again, it turned out in the end that my missing of the "-parse_seqids" option was indeed causing the problem, although I would swear that I tested it before and it wasn't the issue...but anywho, what's important is that it's fixed now.
I noticed a different issue however - if the sequence name of a hit has a comma or a pipe character in it, then you cannot download it's sequence by pressing the FASTA button in the alignment section. Once those are converted to something else (e.g. an underscore), then it works fine. Just thought I'd let you know.
